families/supplychain/python/sawtooth_supplychain/client.py

```diff
@@ -195,9 +194,9 @@ def application_accept(self, record_identifier,
             SupplyChainPayload.APPLICATION_ACCEPT,
             ApplicationAcceptPayload(
                 record_identifier=record_identifier,
-                applicant=applicant,
+                applicant=applicant_public_key,
                 type=application_type,
-                timestamp=timestamp or time.time()),
+                timestamp=timestamp or int(time.time())),
```

Comment:
@cianx should update the spec with the details about what this should be.  I think seconds is fine, if that's what is in the rest.